releasenotes/notes/asm-dependencies-metrics-0f357e2613452be2.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    ASM: dependency telemetry metrics now will only report dependencies actually in use (imported) and will 
```

Comment:
Do we need to make this a `feat` and have a release note? Since telemetry metrics don't have any effect on users?

Reply:
It affects the results on SCA (vulnerable dependencies) on the ASM dashboard so I would prefer to have a changelog in case some users are surprised to see less dependencies listed.